.github/workflows/cron.yml

```diff
@@ -79,48 +88,58 @@ jobs:
         image: redis
     env:
       TEST: bindings
+
     steps:
       - uses: actions/checkout@v2
+
       - name: Set up Python 3.7
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v2
         with:
           python-version: '3.7'
+
       - name: Set up Ruby 2.6
         uses: actions/setup-ruby@v1
         with:
           ruby-version: '2.6'
+
       - name: Install httpie
         run: |
           echo ::group::HTTPIE
           sudo apt-get update -yq
           sudo -E apt-get -yq --no-install-suggests --no-install-recommends install httpie
           echo ::endgroup::
+
       - name: Before Install
         run: |
-          .github/before_install.sh
+          .github/workflows/scripts/before_install.sh
         shell: bash
+
       - name: Install
         run: |
-          .github/install.sh
+          .github/workflows/scripts/install.sh
           echo "HTTPIE_CONFIG_DIR=$GITHUB_WORKSPACE/.github" >> $GITHUB_ENV
           pip install --upgrade pip
         env:
           PY_COLORS: '1'
           ANSIBLE_FORCE_COLOR: '1'
         shell: bash
+
       - name: "Deploy plugin to pypi"
         env:
           PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
         if: "contains(github.ref, 'tags')"
-        run: bash .github/publish_plugin_pypi.sh
+        run: bash .github/workflows/scripts/publish_plugin_pypi.sh
+
       - name: "Publish client to pypi"
         env:
           PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-        run: bash .github/publish_client_pypi.sh
+        run: bash .github/workflows/scripts/publish_client_pypi.sh
+
       - name: "Publish client to rubygems"
         env:
           RUBYGEMS_API_KEY: ${{ secrets.RUBYGEMS_API_KEY }}
-        run: bash .github/publish_client_gem.sh
+        run: bash .github/workflows/scripts/publish_client_gem.sh
+
```

Comment:
We need:
```yaml
- name: Update Redmine
  if: "contains(github.ref, 'tags')"
  run: bash .github/update_redmine.sh
```

Reply:
I'm moving some stuff around - everything that currently has a conditional to check whether there are new tags are being moved into a ```release.yml``` workflow that is explicitly triggered by new tags.